centreon/src/Core/Common/Infrastructure/Repository/ReadVaultRepository.php

```diff
@@ -48,4 +53,27 @@ public function findFromPath(string $path): array
 
         return [];
     }
+
+    public function isConfigurationValid(VaultConfiguration|NewVaultConfiguration $vaultConfiguration): bool
```

Comment:
maybe rename it as testConnection as "valid configuration" is a business consideration